static/include/components/motor-sensor.md

```diff
@@ -2,15 +2,9 @@ If you want to control your motor by specifying the distance and velocity in ter
 
 You can do this in two ways:
 
-The first just uses the movement sensor feedback to increase or decrease the power being set on the motors in increments of 10%.
+The first just uses the encoder feedback to increase or decrease the power being set on the motors in increments of 10%.
 
-The second requires setting the `control_parameters` attribute and uses a PID control loop that measures the current position and change in position and determines the difference or error from the desired distance and velocity to compute a correction for the motor based on three terms:
-
-- A _proportional_ term that is the current error
-- An _integral_ term that is the total cumulative error
-- A _derivative_ term that is the rate of change of the error
-
-By tuning the coefficients on each of these terms, you can adjust how your motor converges towards the target values, how quickly the system reaches the target values, and how much the system overshoots when approaching the target values.
+The second requires setting the `control_parameters` attribute, which sets up a PID control loop to adjust the distance and velocity of the motor. Setting the `control_parameters` will automatically set up the required PID loop for an encoded motor. For more information on PID or to set up a more complex control loop, see the [controls package](../../../docs/internals/controls-package)
```

Comment:
it's complaining about the missing slash and about you having docs in the front I think?
```suggestion
The second requires setting the `control_parameters` attribute, which sets up a PID control loop to adjust the distance and velocity of the motor.
Setting the `control_parameters` will automatically set up the required PID loop for an encoded motor.
For more information on PID or to set up a more complex control loop, see the [controls package](/internals/controls-package/)
```